### ISSUE TITLE
Windows notifications gets on silent mode

### ISSUE THREAD

**Issue by solispauwels** [microsoft/terminal#1840]
# Environment

```none
Windows build number:  10.0.18362.175
Windows terminal version: v0.2.1831.0
```

# Steps to reproduce

- Open windows terminal
- Click on maximize icon

# Expected behavior

Get simply a maximized window

# Actual behavior

The maximized windows causes notifications to be in silent mode (without notification) and does not get out of silent mode even passing to another application.

This is possible related to the unable access tothe task bar issue

**Comment by atruskie** [comment]
So I've noticed this too.

This is an issue with the _Focus Assist_ tool. I believe it is conflating _Full Screen_ with a _Maximized_ window.
![image](https://user-images.githubusercontent.com/1452968/60763023-fcf24400-a0ae-11e9-91c1-c7b95191d34d.png)

If you disable focus assist the issue does not happen. However, I like focus assist...

**Comment by solispauwels** [comment]
In any case when you click on maximize the expected behavior is to get a maximized window, not a full screen window

**Comment by DHowett-MSFT** [comment]
This will also break autohiding taskbars and some other things. There's ... some magic we need to do here like Chromium and VS where they jiggle the window size by 1px to make things "magically work."

**Comment by DHowett-MSFT** [comment]
Link: #1625.

**Comment by ffes** [comment]
From my experience, when I maximize on my primary screen I don't see the silent mode problem. When I maximize on another than the primary screen I do see this issue.

**Comment by ffes** [comment]
To add to my previous comment... It only happens when you disable the taskbar on secondary screens.

So that means is it full screen on a secondary screen and the switch to silent mode is at least explainable. Maybe there is a check for window size vs screen size (just speculating).

Still not what you want. And terminal is the only application I see this behavior.

**Comment by DHowett-MSFT** [comment]
This is the same root cause as /dup #1438, so I'm going to merge them together as "the class of issues about windows thinking we're a game and suppressing shell features."

**Comment by ffes** [comment]
This is was identified and closed as a dup of #1438 which should be fixed in v0.10.761.0, but the case I describe here above in https://github.com/microsoft/terminal/issues/1840#issuecomment-537595876 and https://github.com/microsoft/terminal/issues/1840#issuecomment-553872381 are **NOT** fixed.

Please reopen this issue!

/cc @DHowett-MSFT

**Comment by DHowett-MSFT** [comment]
Thanks!

**Comment by DHowett-MSFT** [comment]
Since it's a fairly minor p3, and we're past the p3 cutoff for version 1 I'm scoping it out, but if somebody were to help us investigate and fix it I'd definitely take the fix for v1.

**Comment by zadjii-msft** [comment]
Okay so this seems crazy, but this seems like it's fixed on Windows 11. Neither maximizing nor fullscreening seems to trigger Focus Assist mode anymore. However, I just checked with my Windows 10 laptop and that _does_ still hit this when fullscreening. 

I'm gonna leave this open, but considering the OS Shell waws able to seemingly fix this on their side, then maybe trying to fix this on our side for Windows 10 might not be the right solution.

**Comment by zadjii-msft** [comment]
Actually, looking through our internal bug tracker, I did find the fix to the OS. That was in fact MSFT:33503732 which is fixed in `22425.1001.rs_prerelease.210720-1539`. The details in that bug also seem to suggest that it's definitely an OS bug, and not something that applies uniquely to the Terminal. I'm gonna mark this as a dupe of that resolved bug. The fix should be out soon<sup>tm</sup>

/dup https://task.ms/33503732
